Packs/GreyNoise/Integrations/GreyNoise/GreyNoise.py

```diff
@@ -1106,6 +1108,150 @@ def context_command(client: Client, args: dict, reliability: str) -> CommandResu
     )
 
 
+@exception_handler
+@logger
+def cve_command(client: Client, args: dict, reliability: str) -> CommandResults:
+    """
+    Returns information about CVE.
+
+    :type client: ``Client``
+    :param client: client object
+
+    :type args: ``dict``
+    :param args: All command arguments, usually passed from ``demisto.args()``.
+    :return: A ``CommandResults`` object that is then passed to ``return_results``,
+           that contains the IP information.
+    :rtype: ``CommandResults``
+
+    :type reliability: ``String``
+    :param reliability: string
+    """
+    cve_arg = args.get("cve", "")
+    cvss = 0
+    description = ""
+    published = ""
+    modified = ""
+    response = client.cve(cve_arg)
+    cve_raw_response = copy.deepcopy(response)
+    response = remove_empty_elements(response)
+    if response.get("id"):
+        cvss = response["details"].get("cve_cvss_score", "N\\A")
+        description = response["details"].get("vulnerability_description", "")
+        vendor = response["details"].get("vendor", "")
+        product = response["details"].get("product", "")
+        if "timeline" in response:
+            published = response["timeline"].get("cve_published_date", "").split("T")[0]
+            modified = response["timeline"].get("cve_last_updated_date", "").split("T")[0]
+        else:
```

Comment:
This is redundant, since in lines 1132-1133 those variables are declared and assignment as "".
So no need for this else